lib/oxidized/model/riverbed.rb

```diff
@@ -0,0 +1,131 @@
+class Riverbed < Oxidized::Model
+  using Refinements
+
+  # Define the prompt
+  PROMPT = /^.*\s*[\w-]+\s*[#>]\s*$/i
```

Comment:
- Directly use `prompt /^.*\s*[\w-]+\s*[#>]\s*$/`
- You don't need case insensitive mode as you don't use characters in the regexp
- use a whitespace instead of \s if you are just matching a whitespace